exercises/transpose/example.lua

```diff
@@ -1,7 +1,7 @@
 local function Grid(s)
   local width = 0
   local rows = {}
-  for row in s:gmatch('([^\n]*)\n?') do
```

Comment:
This works for Lua 5.3.3 and 5.3.1 for me. Is there any reason why we shouldn't use this version instead of `'[^\n]*'`?